server/data/data-hbase/src/test/java/org/apache/james/rrt/hbase/HBaseRecipientRewriteTableTest.java

```diff
@@ -60,4 +61,10 @@ protected AbstractRecipientRewriteTable getRecipientRewriteTable() throws Except
         rrt.configure(new DefaultConfigurationBuilder());
         return rrt;
     }
+
+    @Ignore("JAMES-2376 HBASE RRT getAllMappings is buggy")
+    @Override
+    public void getAllMappingsShouldListAllEntries() {
```

Comment:
can't you fix it quickly ?

Reply:
I'll give it a go tomorrow in a       separate PR?

Reply:
I did a quick fix. But maybe there is a better way to do that with hbase...